src/components/Contacts/ContactListTableRow.jsx

```diff
@@ -14,48 +14,42 @@ import { DocumentListContext } from '@contexts';
 import { StyledTableCell, StyledTableRow } from '../Table/TableStyles';
 
 /**
- * @typedef {import("../../typedefs.js").clientListTableRowProps} clientListTableRowProps
+ * @typedef {import("../../typedefs.js").contactListTableRowProps} contactListTableRowProps
```

Comment:
We could change the typedef in `typedefs.js` as well for `clientListTableRowProps` to `contactListTableRowProps` and their relevant properties.

Reply:
fixed